src/DependencyInjection/LeezyPheanstalkExtension.php

```diff
@@ -22,10 +25,11 @@ class LeezyPheanstalkExtension extends Extension
     public function load(array $configs, ContainerBuilder $container)
     {
         $configuration = new Configuration();
-        $config = $this->processConfiguration($configuration, $configs);
+        $config        = $this->processConfiguration($configuration, $configs);
 
         $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.xml');
+        $loader->load('commands.xml');
 
         if (!$config['enabled']) {
```

Comment:
IMO, you should also update ```CHANGELOG.md```, or even better create an ```UPGRADE-3.0.md``` in order to track this change.